src/rust/uid-allocator/tests/uid_allocator.rs

```diff
@@ -0,0 +1,32 @@
+#![cfg(all(test, feature = "integration_tests"))]
+
+use clap::Parser;
+use rust_proto::graplinc::grapl::api::uid_allocator::v1beta1::{
+    client::UidAllocatorServiceClient,
+    messages::CreateTenantKeyspaceRequest,
+};
+use uid_allocator::client::CachingUidAllocatorServiceClient;
+
+#[tokio::test]
+async fn test_uid_allocator() -> Result<(), Box<dyn std::error::Error>> {
+    let client_config = uid_allocator::config::UidAllocatorClientConfig::parse();
+
+    let tenant_id = uuid::Uuid::new_v4();
+
+    let endpoint = format!("http://{}", &client_config.uid_allocator_connect_address);
+    let mut allocator_client = CachingUidAllocatorServiceClient::new(
+        UidAllocatorServiceClient::connect(endpoint).await?,
+        100,
+    );
```

Comment:
can we maybe provide some prebuilt `fn (UidAllocatorClientConfig) -> CachingUidAllocatorServiceClient` ?

in fact, the `allocation_size` in that ClientConfig is never used, strongly suggesting this function should exist